contribs/gmf/src/directives/print.js

```diff
@@ -2,7 +2,7 @@ goog.provide('gmf.printComponent');
 
 goog.require('gmf');
 /** @suppress {extraRequire} */
-goog.require('gmf.authenticationDirective');
+goog.require('gmf.authentication.module');
```

Comment:
Why do we requested the directive ?
So what you have done is correct, but can't we require only the service ?